.github/workflows/run-help.yml

```diff
@@ -0,0 +1,73 @@
+---
+name: run-help
+'on': [push, pull_request]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  help-linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - arch: 'x86_64'
+            runner: 'ubuntu-latest'
+            suffix: ''
+          - arch: 'arm64'
+            runner: 'ubuntu-24.04-arm'
+            suffix: '-arm64'
+    name: help-linux-${{ matrix.config.arch }}
+    runs-on: ${{ matrix.config.runner }}
+    steps:
+      - name: wait for build
+        uses: lewagon/wait-on-check-action@v1.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          check-name: linux-${{ matrix.config.arch }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 30
+      - name: Download workflow artifact
+        uses: dawidd6/action-download-artifact@v6
+        with:
+          workflow: build.yml
+          name: linux${{ matrix.config.suffix }}
+      - name: Extract artifact
+        run: |
+          chmod +x lmms-*.AppImage
+          ./lmms-*.AppImage --appimage-extract
+      - name: Display help
+        run: |
+          cd squashfs-root/
+          ./AppRun --help | grep "Usage: lmms"
+  help-macos:
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - arch: 'x86_64'
+            runner: 'macos-latest'
+          # ARM is currently disabled because there seems to be no runner available
```

Comment:
macos-latest can run arm64 and intel.

Reply:
So one runner `macos-latest` for both, and I just choose different artifacts?

Reply:
Yes, that should work.

Reply:
Nice, it worked indeed :+1:

Reply:
@JohannesLorenz this comment about "no runner available" is still in the code.  Did you mean to remove it?

Reply:
I'm dumb. I pushed my fix to the upstream remote, leaving this unchanged. Please check again.

Reply:
Looks good, thanks!